CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+### Functionality
+#### OTTR_Template
+
+The [OTTR_Template repository](https://github.com/jhudsl/OTTR_Template) is a template used to create OTTR courses, specifically when you're writing one course, but publishing it in three places: (bookdown/website), Leanpub, and Coursera. If used with [a pull request model](https://www.ottrproject.org/writing_content_courses.html#about-ottr-and-pull-requests), courses built with the template will be checked for spelling errors and periodically for broken URLs, and can be previewed and automatically rendered after updates.
+
+#### OTTR_Quizzes
+
+The [OTTR_Quizzes repository](https://github.com/jhudsl/OTTR_Quizzes) can be used when you want your course quizzes to remain private rather being housed in a public, open source repository. Use of this repository is in conjunction with the main OTTR_Template. All edits and additions to non-quiz content should take place
+
+#### OTTR_Template_Website
+
+The [OTTR_Template_Website repository](https://github.com/jhudsl/OTTR_Template_Website) is a template used to create OTTR websites. Like the main OTTR_Template, if used with [a pull request model](https://www.ottrproject.org/writing_content_courses.html#about-ottr-and-pull-requests), websites built with the template will be checked for spelling errors and periodically for broken URLs, and can be previewed and automatically rendered after updates. Unlike the main OTTR_Template, it only produces a website as output.
+
+#### ottr-reports
+
+The [ottr-reports repository](https://github.com/jhudsl/ottr-reports) houses the functionality for reporting the results of the GitHub Action checks on spelling, URLs, and quiz formatting.
+
+#### ottrpal
+
+The [ottrpal repository](https://github.com/jhudsl/ottrpal) provides the functionality to convert an OTTR course to files ready for use in Leanpub and Coursera
+
+#### cow
+
+The [cow repository](https://github.com/jhudsl/cow) isn't officially an OTTR repository, but is a GitHub API wrapper in R which has additional functions to help wrangle GitHub published courses like OTTR courses.
+
+### Documentation
+
+#### ottrproject.org
+
+The [ottrproject.org repository](https://github.com/jhudsl/ottrproject.org) renders the [OTTR documentation website](https://www.ottrproject.org/). The repository is generated from the [OTTR_Template_Website template](https://github.com/jhudsl/OTTR_Template_Website).
+
+#### Wiki
+
+Listed as the "main OTTR guide" on some READMEs, there's an [OTTR wiki](https://github.com/jhudsl/OTTR_Template/wiki).
```

Comment:
Ah yes. We've been working on slowly deprecating this and will archive it eventually. Is there anywhere you found that is still pointing to this wiki? If so we should get rid of those and also not mention it in this CONTRIBUTING.md guide. Thanks.

Reply:
Removed in [commit 5208345](https://github.com/jhudsl/OTTR_Template/pull/731/commits/520834517635eb81cb6af79c926d946f84355712). Will remove from ottrpal README.

Reply:
Removed from ottrpal README in [commit bb3bb6c](https://github.com/jhudsl/ottrpal/pull/125/commits/bb3bb6c7799e17e0a251a7dc5dbe837ea48107ec)